actioncable/lib/action_cable.rb

```diff
@@ -0,0 +1,50 @@
+#--
+# Copyright (c) 2015 Basecamp, LLC
```

Comment:
Was following the pattern from https://github.com/rails/rails/blob/master/actionmailer/lib/action_mailer.rb. Don't have a strong opinion either way, but we should be consistent.